lib/moonshot/controller_config.rb

```diff
@@ -21,5 +25,17 @@ def initialize
       @plugins = []
       @show_all_stack_events = false
     end
+
+    def parameter_strategy=(value)
```

Comment:
I'm gonna outsource it to the CLI class, but this way please note that using Moonshot without the CLI means the user not only _could_, but _**must**_ bring their own parameter strategy.